streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -291,23 +293,35 @@
      * <p>
      * Enabling exactly-once processing semantics requires broker version 0.11.0 or higher.
      * If you enable this feature Kafka Streams will use more resources (like broker connections)
-     * compared to the {@link #AT_LEAST_ONCE} case.
+     * compared to {@link #AT_LEAST_ONCE "at_least_once"} and {@link #EXACTLY_ONCE_V2 "exactly_once_v2"}.
      *
-     * @see #EXACTLY_ONCE_BETA
+     * @deprecated Since 3.0.0, will be removed in 4.0. Use {@link #EXACTLY_ONCE_V2 "exactly_once_v2"} instead.
```

Comment:
See comment above -- I'm ok with removing this for the `sendOffsetsToTransaction` since all users have to do there is update one line of code, but for eos that may require a broker upgrade I'd rather leave this in to encourage them to do so